src/parsing/parsers/generic/genericAutocompleteParser.test.ts

```diff
@@ -35,7 +35,23 @@ import { extractTestCases, runTestCases } from '../../test/testing';
 import { assertPartials, CommonParser } from '../../lib/parsing-typed';
 
 const jisonFolder = 'src/parsing/parsers/generic/jison';
-const groupedTestCases = extractTestCases(jisonFolder, structure.autocomplete);
+
+let structureFiles = structure.autocomplete
+// We slice first 3 arguments which required for the other places
+const exactFiles = process.argv.slice(3).filter((arg) => /.jison$/.test(arg))
+if (exactFiles.length !== 0) {
+  structureFiles = structureFiles.filter(
+      (filePath) => exactFiles.some(
+          (exactFile) => new RegExp(exactFile + '$').test(filePath)
+      )
+  )
+}
+
+const groupedTestCases = extractTestCases(jisonFolder, structureFiles);
+
+if (groupedTestCases.length === 0) {
+  process.stdout.write(''.concat('\x1b[33m', 'JISON test cases list is empty', '\x1b[0m\ ', '\n'));
+}
```

Comment:
We don't need this filtering code.

You can list all the tests with `jest --verbose`, you'll see a list like this:
```
    truncate/truncate_table.jison
      ✓ should not report errors given "TRUNCATE TABLE baa.boo;|"
      ✓ should suggest keywords given "truncate |"
      ✓ should suggest tables given "TRUNCATE TABLE |" (1 ms)
    update/update_table.jison
      ✓ should suggest keywords given "|"
      ✓ should suggest keywords given "UPDATE bar  |"
      ✓ should suggest keywords given "UPDATE bar SET id=1, foo=2 |"
      ✓ should suggest keywords given "UPDATE bar SET id |"
      ✓ should suggest tables given "UPDATE |" (1 ms)
      ✓ should suggest tables given "UPDATE bla|"
      ✓ should suggest tables given "UPDATE bar.|"
      ✓ should suggest tables given "UPDATE bar.foo|"
      ✓ should suggest columns given "UPDATE bar.foo SET |"
      ✓ should suggest columns given "UPDATE bar.foo SET id = 1, bla = 'foo', |" (1 ms)
      ✓ should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE |"
      ✓ should suggest values given "UPDATE bar.foo SET bla = 'foo' WHERE id = |" (1 ms)
      ✓ should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE id = 1 AND |"
    use/use.jison
      ✓ should suggest keywords given "|"
      ✓ should suggest databases given "USE |"
      ✓ should suggest databases given "USE bla|"
      ✓ should use a use statement given "use database_two; \nSELECT |" (1 ms)
      ✓ should use the last use statement given "USE other_db; USE closest_db; \n       SELECT |"
      ✓ should use the use statement given "USE other_db; USE closest_db; \n    SELECT |; USE some_other_db;"
```

And then you can filter by name with -t like so `jest -t 'update/update_table.jison' --verbose`
```
    truncate/truncate_table.jison
      ○ skipped should not report errors given "TRUNCATE TABLE baa.boo;|"
      ○ skipped should suggest keywords given "truncate |"
      ○ skipped should suggest tables given "TRUNCATE TABLE |"
    update/update_table.jison
      ✓ should suggest keywords given "|" (15 ms)
      ✓ should suggest keywords given "UPDATE bar  |" (1 ms)
      ✓ should suggest keywords given "UPDATE bar SET id=1, foo=2 |" (1 ms)
      ✓ should suggest keywords given "UPDATE bar SET id |" (1 ms)
      ✓ should suggest tables given "UPDATE |"
      ✓ should suggest tables given "UPDATE bla|"
      ✓ should suggest tables given "UPDATE bar.|" (1 ms)
      ✓ should suggest tables given "UPDATE bar.foo|"
      ✓ should suggest columns given "UPDATE bar.foo SET |"
      ✓ should suggest columns given "UPDATE bar.foo SET id = 1, bla = 'foo', |"
      ✓ should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE |"
      ✓ should suggest values given "UPDATE bar.foo SET bla = 'foo' WHERE id = |" (1 ms)
      ✓ should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE id = 1 AND |" (1 ms)
    use/use.jison
      ○ skipped should suggest keywords given "|"
      ○ skipped should suggest databases given "USE |"
      ○ skipped should suggest databases given "USE bla|"
      ○ skipped should use a use statement given "use database_two; \nSELECT |"
      ○ skipped should use the last use statement given "USE other_db; USE closest_db; \n       SELECT |"
      ○ skipped should use the use statement given "USE other_db; USE closest_db; \n    SELECT |; USE some_other_db;"
```

You can also run solo test like so `jest -t 'should not report errors given "TRUNCATE TABLE baa.boo;' --verbose`
```
    truncate/truncate_table.jison
      ✓ should not report errors given "TRUNCATE TABLE baa.boo;|" (17 ms)
      ○ skipped should suggest keywords given "truncate |"
      ○ skipped should suggest tables given "TRUNCATE TABLE |"
    update/update_table.jison
      ○ skipped should suggest keywords given "|"
      ○ skipped should suggest keywords given "UPDATE bar  |"
      ○ skipped should suggest keywords given "UPDATE bar SET id=1, foo=2 |"
      ○ skipped should suggest keywords given "UPDATE bar SET id |"
      ○ skipped should suggest tables given "UPDATE |"
      ○ skipped should suggest tables given "UPDATE bla|"
      ○ skipped should suggest tables given "UPDATE bar.|"
      ○ skipped should suggest tables given "UPDATE bar.foo|"
      ○ skipped should suggest columns given "UPDATE bar.foo SET |"
      ○ skipped should suggest columns given "UPDATE bar.foo SET id = 1, bla = 'foo', |"
      ○ skipped should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE |"
      ○ skipped should suggest values given "UPDATE bar.foo SET bla = 'foo' WHERE id = |"
      ○ skipped should suggest columns given "UPDATE bar.foo SET bla = 'foo' WHERE id = 1 AND |"
    use/use.jison
      ○ skipped should suggest keywords given "|"
      ○ skipped should suggest databases given "USE |"
      ○ skipped should suggest databases given "USE bla|"
      ○ skipped should use a use statement given "use database_two; \nSELECT |"
      ○ skipped should use the last use statement given "USE other_db; USE closest_db; \n       SELECT |"
      ○ skipped should use the use statement given "USE other_db; USE closest_db; \n    SELECT |; USE some_other_db;"
```

Reply:
I don't like this approach because we can use built-in tools for it. Also this one will work incorrectly if we'll run `jest` with multiple flags. So this code piece just breaks lots of useful functionality, so let's rollback it.